frontend/__tests__/src/fixtures.ts

```diff
@@ -18,6 +18,10 @@ export const RESET = 'Reset'
 
 export const VERIFIED = 'Verified'
 
+export const mockInfo = 'mockToken'
```

Comment:
Constant?

Reply:
have changed this constant to MOCK_INFO

Reply:
I'm sorry to misunderstand this problem, I'll create a new PR to solve it soon.